tutorials/vpc-private-gateway-autoconfiguration/index.mdx

```diff
@@ -0,0 +1,202 @@
+
+## Creating a Public Gateway
+
+1. Click the [Public Gateways](https://console.scaleway.com/vpc/public-gateways) tab in the VPC section of the Scaleway console. The Public Gateways splash screen displays. Click **Create a Public Gateway**:
+
+  <Lightbox src="scaleway_vpc_public_gateway_splash.png" alt="" />
+
+2. The Private Network creation wizard displays. Enter the details of the new Public Gateway: 
+
+  - Choose the availability Zone of the Public Gateway.
+  - Select the Public Gateway offer
+  - Select the Public Gateway IP address. To create a new IP, select **Allocate new IP** in the drop-down list. 
+  - Enter a name and optional tags for the Private Gateway
+
+3. Click **Create a Public Gateway**. The list of your Public Gateways displays.
+
+  <Lightbox src="scaleway_vpc_public_gateway_list.png" alt="" />
+
+4. Click **Manage** to display the Public Gateway Information. 
+
+5. Click **+ Attach a new Private Network** to link the previously created Private Network with the Public Gateway.
+
+6. Select the Private Network from the drop-down list, enable the DHCP server and dynamic NAT. Then click **Attach Private Network**:
+
+  <Lightbox src="scaleway_vpc_public_gateway_attach_network.png" alt="" /> 
+
+7. Check if the IP address for the Private Network has been assigned to the instance using the DHCP server of the Private Gateway by running the `ip a` command: 
+
+  ```
+  3: ens5: <BROADCAST,MULTICAST,UP,LOWER_UP> mtu 1500 qdisc fq_codel state UP group default qlen 1000
+    link/ether 02:00:00:00:5a:d8 brd ff:ff:ff:ff:ff:ff
+    inet 192.168.0.2/24 brd 192.168.0.255 scope global dynamic ens5
+       valid_lft 3513sec preferred_lft 3513sec
+    inet6 fe80::ff:fe00:5bea/64 scope link 
+       valid_lft forever preferred_lft forever
+  ```
+
+  You can see the private IP address `192.168.0.2` has been configured automatically on the interface.
+
+## Adding a second Instance to the Private Gateway
+
+1. Click **Instances** in the **Compute** section of the Scaleway console to display the list of your instances. Click **+ Create an Instance**.
+
+2. Enter the details for your new instance and click **Create a new Instance**. The instace is being created. Once this is done, the list of your Instances displays.
+
+3. Click **VPC** in the **Network** section. The list of your Private Networks displays.
+
+4. Click <icon name="more" /> > **More info** to display the details of the Private Network.
+
+5. Click the **Attached Ressources** tab to display a list of resources attached to your Private Network.
+
+7. Select your new Instance in the drop down list and click **Add Instance** to attach it to the Private Network:
+
+  <Lightbox src="scaleway_vpc_add_instance.png" alt="" />
+
+  The new Instance displays in the list of devices connected to the Private Network.
+
```

Comment:
I don't think this section is required in the tutorial. It doesn't bring anything new, does it?

```suggestion
```